src/modules/netinstall/README.md

```diff
@@ -0,0 +1,28 @@
+# Netinstall module
+
+The netinstall module allows distribution maintainers to ship minimal ISOs with only a basic set of preinstall packages. At installation time, the user is presented with the choice to install groups of packages from a predefined list.
+
+Calamares will then invoke the correct backend to install the packages.
+
+## How are packages configured?
+The *netinstall.conf* file should have this format:
+
+    ----
+    groupsUrl: <URL to YAML file>
```

Comment:
Having a remote file listing the packages will reduce the number of respin that need to be done because a hardcoded pkg name on the ISO doesn't exist anymore and the ISO has become useless.

Maybe the file should be signed and Calamares should verify that signature?